Kernel/RTC.cpp

```diff
@@ -35,8 +36,27 @@ static u8 bcd_to_binary(u8 bcd)
 
 void read_registers(unsigned& year, unsigned& month, unsigned& day, unsigned& hour, unsigned& minute, unsigned& second)
 {
-    while (update_in_progress())
-        ;
+    // Note: Let's wait a 0.1 second until we stop trying to query the RTC CMOS
```

Comment:
Small typo made here, this should be ``// Note: Let's wait 0.01 seconds until we stop...``

Reply:
Thank you! :)